Formula/iota.rb

```diff
@@ -22,6 +22,9 @@ class Iota < Formula
         end
     end
 
+    # Return with error if no compatible architecture was found.
+    odie "Unsupported architecture. Please use cargo install and build from source" if arch == ""
```

Comment:
Please also include the arch in the error message, may make it easier to debug

Reply:
```suggestion
    odie "Unsupported architecture (#{RUBY_PLATFORM}). Please use cargo install and build from source" if arch == ""
```

Reply:
Not sure if this is the best method but in my case it prints something like:
`Error: Unsupported architecture (x86_64-darwin15). Please use cargo install and build from source`